new-standalone-service.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+cleanup() {
+    # TODO(mohit): Maybe add a trap for SIGINT.
+    rm -rf template.zip
+}
+
+info() {
+    MESSAGE=$1
+    echo $MESSAGE
+}
+
+debug() {
+    if [[ -z "${DEBUG// }" ]]; then
+        return
+    fi
+    MESSAGE=$1
+    echo "DEBUG: $MESSAGE"
+}
+
+PROJECT_NAME=$1
+PROJECT_PATH=$2
+
+if [[ -z "${PROJECT_NAME// }" ]]; then
+    echo "You must provide the name of the project as the first argument"
+    echo "Usage: ./new-standalone-service.sh project-name"
+    echo "Example: ./new-standalone-service.sh kafka"
+    cleanup
+    exit 1
+fi
+
+if [[ -z "${PROJECT_PATH// }" ]]; then
+    PROJECT_PATH=$(pwd)
+fi
+
+debug "Scaffolding $PROJECT_NAME from template"
+
+cp -R frameworks/template $PROJECT_PATH/$PROJECT_NAME
+cp -R tools $PROJECT_PATH/$PROJECT_NAME/tools
+
+rm -rf $PROJECT_PATH/$PROJECT_NAME/build
+rm -rf $PROJECT_PATH/$PROJECT_NAME/cli/dcos-*/*.whl
+rm -rf $PROJECT_PATH/$PROJECT_NAME/cli/dcos-*/dcos-*
+rm -rf $PROJECT_PATH/$PROJECT_NAME/cli/python/{build,dist}
+rm -rf $PROJECT_PATH/$PROJECT_NAME/build.sh
+
+cat > $PROJECT_PATH/$PROJECT_NAME/build.sh <<'EOF'
+#!/bin/bash
+set -e
+
+# capture anonymous metrics for reporting
+curl https://mesosphere.com/wp-content/themes/mesosphere/library/images/assets/sdk/build-sh-start.png >/dev/null 2>&1
+
+FRAMEWORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+BUILD_DIR=$FRAMEWORK_DIR/build/distributions
+PUBLISH_STEP=${1-none}
+export REPO_NAME="$(basename $FRAMEWORK_DIR)"
+export BUILD_BOOTSTRAP=no
+export TOOLS_DIR=${FRAMEWORK_DIR}/tools
+export CLI_DIR=${FRAMEWORK_DIR}/cli
+export ORG_PATH=github.com/$REPO_NAME
```

Comment:
wouldn't this result in e.g. `ORG_PATH=github.com/dcos-commons` instead of `ORG_PATH=github.com/mesosphere`?

also it looks like this is overridden in `tools/build_go_exe.sh` so it might not actually be doing anything